src/imgtools/io/loaders.py

```
@@ -154,19 +175,22 @@
-                 study_names=[],
-                 series_names=[],
-                 subseries_names=[],
-                 id_column=None,
-                 expand_paths=False,
-                 readers=None):
-
+    def __init__(
+        self,
+        json_path,
+        csv_path_or_dataframe,
+        col_names=[],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)